requirements/requirements-dev.txt

```diff
@@ -53,8 +53,8 @@ pylint==2.2.2
 pyslack==0.4.0
 pytest-asyncio==0.10.0
 pytest==4.2.0
+python-dateutil==2.8.0
 python-magic==0.4.15
-python_dateutil==2.8.0
```

Comment:
@greysteil
Hey, I just noticed an inconsistency with dashes and underscores in names.
`python-dateutil` has been replaced by `python_dateutil` just a week ago by https://github.com/das7pad/hangoutsbot/pull/120.

Reply:
I think this is going to be due to the version of `pip-compile` that Dependabot is using. My hunch is that never versions of `pip-compile` normalise dependency names according to [PEP 503](https://www.python.org/dev/peps/pep-0503/#normalized-names).

Reply:
`pip-compile` in `python/helpers/requirements.txt` was updated two weeks+ ago.

Reply:
Hmmm, digging into it it looks like the underscore was introduced [here](https://github.com/das7pad/hangoutsbot/commit/0f47a857f670c162775d00e6ef4317f4a114c702#diff-444e80f30bd2f1e25b779966c9621b9b). Let me take a look into what happened there...

Reply:
Just spent a couple of hours digging into it. I *think* it's fixed on master of pip-tools, as I can't replicate it when running that but I *can* replicate it when running v3.3.2. Let's check again after the next pip-tools release.

Reply:
Thank you for your work on this issue!

I took a look at the master..v3.3.2 diff and the changes were documentation/verbose handling related. Nothing that should change internal logic. Next up are the changes in `pip` for `19.0.1..19.0.2`, which was updated [recently](https://github.com/dependabot/dependabot-core/commit/484e3f7570a5ac6694e21826c60b0fd2e64be7dd). The only change related to names is [here](https://github.com/pypa/pip/compare/19.0.1...19.0.2#diff-79304b3f74b6bf37043da4d96ccd3755R868). I am not sure yet, how it could tamper our name.